src/backend.py

```diff
@@ -155,3 +156,11 @@ def handle_status_code(status_code):
     async def get_wow_character_achievements(self,  realm, character_name):
         url = f"https://{self._authentication_client.region}.api.blizzard.com/wow/character/{realm.lower()}/{character_name}?fields=achievements"
         return await self.do_request("GET", url)
+
+    async def get_ow_player_data(self):
+        url = 'https://owapi.io/profile/pc/' + self._authentication_client.region + '/' + self._authentication_client.user_details['battletag'].replace('#', '-')
+        response = await self.do_request('GET', url, None, False)
+        payload = json.loads(response.text)
```

Comment:
Isn't it always json reponse? Then just:
```suggestion
        payload = await self.do_request('GET', url, json=True)
```